app/src/main/java/swati4star/createpdf/adapter/ViewFilesAdapter.java

```diff
@@ -78,7 +82,10 @@ public void onBindViewHolder(@NonNull ViewFilesHolder holder, final int position
         Log.d("logs", "getItemCount: " + mFileList.size());
         // Extract file name from path
         final String fileName = mFileList.get(position).getPath();
-        String[] name = fileName.split("/");
+        final String[] name = fileName.split("/");
+        File file = mFileList.get(position);
+        final Date lastModDate = new Date(file.lastModified());
```

Comment:
We have a function to get last modified date also : https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/java/swati4star/createpdf/util/FileUtils.java#L134

If the format returned by this function is different than yours, create a new one.